examples/hls_sink.exs

```diff
@@ -42,45 +42,44 @@ defmodule Example do
   def handle_init(_) do
     children = [
       source: %Membrane.Hackney.Source{
-        location: "https://raw.githubusercontent.com/membraneframework/static/gh-pages/video-samples/test-video.h264",
+        location:
+          "https://raw.githubusercontent.com/membraneframework/static/gh-pages/video-samples/test-video.h264",
         hackney_opts: [follow_redirect: true]
       },
       parser: %Membrane.H264.FFmpeg.Parser{
         framerate: {30, 1},
         alignment: :au,
         attach_nalus?: true
       },
-      payloader: Membrane.MP4.Payloader.H264,
-      cmaf_muxer: %Membrane.MP4.CMAF.Muxer{
-        segment_duration: 2 |> Membrane.Time.seconds()
-      },
-      sink: %Membrane.HTTPAdaptiveStream.Sink{
+      sink_stream: %Membrane.HTTPAdaptiveStream.StreamSinkBin{
+        muxer_segment_duration: 2 |> Membrane.Time.seconds(),
         manifest_module: Membrane.HTTPAdaptiveStream.HLS,
         target_window_duration: 30 |> Membrane.Time.seconds(),
         target_segment_duration: 2 |> Membrane.Time.seconds(),
         persist?: false,
-        storage: %Membrane.HTTPAdaptiveStream.Storages.FileStorage{directory: System.get_env("HLS_OUTPUT_DIR", "output")}
+        storage: %Membrane.HTTPAdaptiveStream.Storages.FileStorage{
+          directory: System.get_env(__DIR__)
```

Comment:
Are you sure that there will be an environmental variable called `.`? Didn't you mean to put something like this?
```elixir
directory: __DIR__
```